packages/sdk/src/sendPanic.ts

```diff
@@ -79,8 +78,7 @@ export async function sendPanic(error: RustPanic, cliVersion: string, engineVers
       dbVersion: dbVersion,
     }
 
-    // TODO: What is the exports doing here? A jest thing?
```

Comment:
Was it a jest thing in the end?

Reply:
More accurately, it was something to expose a function to jest in order to mock it (for testing `sendPanic`)